app/javascript/components/gtl/DataTable.jsx

```diff
@@ -60,13 +67,16 @@ export const DataTable = ({
       <tr>
         {!inEditMode() && !noCheckboxes() &&
         <th className="narrow table-view-pf-select">
+          <label className="hiddenCheckboxLabel" id="selectAll" aria-hidden="true">selectAll</label>
           {selectAll()}
         </th>
         }
         {columns.map((column, index) =>
           (noCheckboxes() || inEditMode() || (index !== 0 && !noCheckboxes())) &&
             <th
-              onClick={() => onSort({headerId: column.col_idx, isAscending: settings.sort_dir == "ASC" })}
+              onClick={onSort({headerId: column.col_idx, isAscending: settings.sort_dir == "ASC" })}
+              onKeyPress={onSort({headerId: column.col_idx, isAscending: settings.sort_dir == "ASC" })}
```

Comment:
Duplicate function definition...

Reply:
that's a call to [onSort](https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/javascript/components/gtl-view.jsx#L380)